setup.cfg

```diff
@@ -55,6 +55,7 @@ where = src
 
 [options.extras_require]
 testing =
+    setuptools >= 50
```

Comment:
Why do you need this? We do not have any hard dependency in tests on setuptools, do we?

And FWIW, we typically use this base repo and we merge it for the boilerplate files: https://github.com/nexB/skeleton

In all cases we are progressively moving away from setuptools to use a new build tool called flot https://github.com/nexB/flot that is grown from flit.

Reply:
The `pytest` output without `setuptools`, specifically, is:

```text
============================= test session starts ==============================
platform linux -- Python 3.12.3, pytest-8.2.2, pluggy-1.5.0
rootdir: /var/tmp/saneyaml
configfile: pyproject.toml
plugins: xdist-3.6.1
collected 24 items / 1 error                                                   

==================================== ERRORS ====================================
__________________________ ERROR collecting setup.py ___________________________
ImportError while importing test module '/var/tmp/saneyaml/setup.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
setup.py:3: in <module>
    import setuptools
E   ModuleNotFoundError: No module named 'setuptools'
=========================== short test summary info ============================
ERROR setup.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.35s ===============================
```

Reply:
(Looks like it's picking it up because of this:)

https://github.com/nexB/saneyaml/blob/40e5fa7c0b6e0012452053839184e5cd29802063/pyproject.toml#L10
https://github.com/nexB/saneyaml/blob/40e5fa7c0b6e0012452053839184e5cd29802063/pyproject.toml#L43

Reply:
Anyway, I'll drop that commit, since it only affects local testing.

Reply:
An exclusion in setup.cfg looks best indeed.